flocker/cli/script.py

```diff
@@ -60,8 +63,12 @@ class DeployOptions(Options):
                 "<control-host> <deployment.yml-path> <application.yml-path>"
                 "{feedback}").format(feedback=FEEDBACK_CLI_TEXT)
 
-    optParameters = [["port", "p", REST_API_PORT,
-                      "The REST API port on the server.", int]]
+    optParameters = [
+        ["port", "p", REST_API_PORT,
+         "The REST API port on the server.", int],
+        ["certificate-path", "c",
+         None, "Certificate path, defaults to current directory."],
```

Comment:
And maybe give the parameter a name that suggests it's a directory; `certificates-directory` or something.